slack/frontend/package.json

```diff
@@ -18,100 +18,97 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^16.2.0",
-    "@angular/cdk": "^16.2.0",
-    "@angular/common": "^16.2.0",
-    "@angular/compiler": "^16.2.0",
-    "@angular/core": "^16.2.0",
-    "@angular/elements": "^16.2.0",
-    "@angular/forms": "^16.2.0",
-    "@angular/localize": "^16.2.0",
-    "@angular/platform-browser": "^16.2.0",
-    "@angular/platform-browser-dynamic": "^16.2.0",
-    "@angular/router": "^16.2.0",
-    "@mdi/font": "7.2.96",
+    "@angular/animations": "^17.2.2",
```

Comment:
There is another package.json which still needs the angular upgrade to 17

Reply:
Which?